jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java

```diff
@@ -298,6 +301,24 @@ public int getAcceptors()
         return _acceptors.length;
     }
 
+    /**
+     * @return Returns the optional local name override
+     */
+    @ManagedAttribute("local name")
+    public String getLocalName()
+    {
+        return _localName;
+    }
+
+    /**
+     * Optional override of connection local name used within application API layer
+     * when identifying the local host name of a connected endpoint.
+     */
+    public void setLocalName(String localName)
```

Comment:
Do we just need a `localName` or we actually need a `localAuthority` (i.e. host _and_ port)?
@gregw thoughts?

Reply:
authority is better

Reply:
Still on AbstractConnector?
As in `AbstractConnector.setLocalAuthority(String authority)`?

Reply:
Changed to use authority instead.
Host AND port must be specified though.